ReactNative/ReactTwitter/__tests__/core/service/oauth-helper.test.js

```diff
@@ -3,6 +3,28 @@ jest.unmock('../../../core/service/percent-encoder.js')
 // var OauthHelper = require('../../../core/service/oauth-helper.js')
 
 describe('OauthHelper', () => {
+  // describe('URL Callback', () => {
```

Comment:
Have you opened a bug on the react-native repo regarding the impossibility to test modules that require react-native components?